src/command-line/uninstall.js

```diff
@@ -0,0 +1,72 @@
+"use strict";
+
+const colors = require("colors/safe");
+const path = require("path");
+const program = require("commander");
+const Helper = require("../helper");
+const Utils = require("./utils");
+
+program
+	.command("uninstall <package>")
+	.description("Uninstall a theme or a package")
+	.on("--help", Utils.extraHelp)
+	.action(function(packageName) {
+		const fs = require("fs");
+		const child = require("child_process");
+
+		if (!fs.existsSync(Helper.getConfigPath())) {
```

Comment:
There's a lot of similar code to install here that could be abstracted out, but I have similar changes in an update command, so I'm happy to wait till that's in if you are.

Reply:
Yeah let's have a cleanup-specific PR then, so that feature-PRs don't have to contain lots of not-directly-related changes. Easier to review and to check history later.